modules/core/src/test/java/org/apache/ignite/internal/processors/database/BPlusTreeReplaceRemoveRaceTest.java

```diff
@@ -352,53 +347,24 @@ public TestPairTree(
      */
     @Test
     public void testConcurrentPutRemove() throws Exception {
-        for (int i = 0; i < 50; i++) {
-            TestPairTree tree = new TestPairTree(
-                null,
-                true,
-                CACHE_ID,
-                pageMem,
-                allocateMetaPage().pageId(),
-                lockTrackerManager
-            );
-
-            tree.putx(new Pair(1, 0));
-            tree.putx(new Pair(2, 0));
-            tree.putx(new Pair(4, 0));
-            tree.putx(new Pair(6, 0));
-            tree.putx(new Pair(7, 0));
-
-            // Split root.
-            tree.putx(new Pair(5, 0));
-
-            // Split its left subtree.
-            tree.putx(new Pair(3, 0));
+        for (int i = 0; i < 100; i++) {
```

Comment:
With number of iterations please fix javadoc as well. This number is mentioned there as well.